src/components/Modal/PopupPage.tsx

```diff
@@ -0,0 +1,37 @@
+import { FC, useState } from 'react'
+import CheckRounded from '@mui/icons-material/CheckRounded'
+import IconButton from '@mui/material/IconButton'
+import CloseRoundedIcon from '@mui/icons-material/CloseRounded'
+import Container from '@/layouts/Container'
+
+const PopupPage: FC = () => {
+  const [isOpen, setIsOpen] = useState<boolean>(true)
+  const handleClose = (): void => {setIsOpen(false)}
+  if (!isOpen) {
+    return null
+  }
+  return (
+    <Container>
+      <div className="justify-center items-center">
+        <div className="border rounded-md w-96 h-80 bg-white max-auto text-center">
+          <div className="text-end">
+            <IconButton onClick={handleClose} className="text-black w-12 h-12">
+              <CloseRoundedIcon className="text-[32px]" />
+            </IconButton>
+          </div>
+          <div className="flex justify-center">
+            <div className="bg-[#0EA739] w-20 h-20 rounded-full">
+              <IconButton>
+                <CheckRounded className="text-white text-[64px]" />
+              </IconButton>
+            </div>
+          </div>
+          <div className="text-center font-semi-bold text-xl mt-12">
+           Sign Up Successfully
+          </div>
+        </div>
+      </div>
+    </Container>
```

Comment:
```suggestion
        <Container>
      <div className="border rounded-md w-[494px] h-[405px] mx-auto text-center">
        <div className="text-end pr-5 pt-5 mb-8">
          <CloseRoundedIcon className="text-4xl cursor-pointer" onClick={() => setIsOpen(false)} />
        </div>

        <CheckRounded className="bg-[#0EA739] rounded-full text-white text-[99px] mb-12" />
        <div className="font-semibold text-2xl">Sign Up Successfully</div>
      </div>
    </Container>
```
I just tried rewrite something similar to yours, **NEVER** use flex when you can achieve the same result